controllers/tenants.rb

```diff
@@ -1,9 +1,42 @@
 # index
-get '/apartments/:id/tenants' do
-  erb :"tenants/index"
+get '/tenants' do
+    @tenants = Tenants.all
+    erb(:"tenants/index")
+end
+
+# new
+get "/tenants/new" do
+  erb(:"tenants/new")
+end
+
+#create
+post "/tenants" do
+  @tenant = Tenant.create!(params[:tenant])
+  redirect("/tenants/#{@tenant.id}")
 end
 
 # show
-get '/apartments/:id/tenants/new' do
-  erb :"tenants/new"
+get '/tenants/:id' do
+  @tenant = tenant.find(params[:id])
+  erb(:"tenants/show")
+end
```

Comment:
See how tenants was nested under a specific apartment?  This is the first step toward listing the tenants in a specific apartment.